libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json

```diff
@@ -312,6 +312,11 @@
         "comment": "Message of the scheduled email.",
         "limit": 0
     },
+    "dialogs.schedule.email.unsubscribed.recipients": {
+        "value": "{n, plural, one {1 recipient unsubscribed from this schedule} other {# recipients unsubscribed from this schedule}}.",
```

Comment:
I would do `{n, plural, one {1 recipient unsubscribed} other {# recipients unsubscribed}} from this schedule.`